libraries/AP_Vehicle/AP_Vehicle.cpp

```diff
@@ -635,7 +634,7 @@ void AP_Vehicle::update_throttle_notch(AP_InertialSensor::HarmonicNotch &notch)
     const float motors_throttle = motors != nullptr ? abs(motors->get_throttle() / 100.0f) : 0;
 #endif
 
-    float throttle_freq = ref_freq * MAX(min_ratio, sqrtf(motors_throttle / ref));
+    float throttle_freq = ref_freq * sqrtf(MAX(0,motors_throttle) / ref);
```

Comment:
I don't think we need this `MAX`. 631 and 634 will always be >= 0

Reply:
ok, I was thinking about things like sub, and wasn't absolutely sure we couldn't get a negative number. The existing code did protect against negative numbers here, so I kept that property. We can certainly remove it if there is no way for it to happen

Reply:
the freq_min_ratio is now handled in HarmonicNotchFilter<T>::set_center_frequency(), via _minimum_freq which is calculated in the init() call